getting-started/troubleshooting.html.md

```diff
@@ -155,3 +155,18 @@ Use these two commands to see environment variables.
 flyctl secrets list
 flyctl config env
 ```
+
+## _Is your buildpack-based app failing to deploy?_
+
+First of all, we think using a [Dockerfile](https://fly.io/docs/languages-and-frameworks/dockerfile/) rather than buildpacks is more reliable and faster to deploy.
+
+That's because buildpacks come with lots of dependencies to be able to build different app stacks rather than just what you need. On top of that, we've seen buildpack providers upgrade the image on dockerhub and things Stop Working (even with no code changes on your app). Fly launch already generates Dockerfiles for [many](https://fly.io/ruby-dispatch/rails-on-docker/) [popular](https://fly.io/docs/elixir/getting-started/#generate-the-app-and-deploy-with-postgres) [frameworks](https://fly.io/docs/django/getting-started/#provision-django).
```

Comment:
```suggestion
That's because buildpacks come with lots of dependencies to be able to build different app stacks rather than just what you need. On top of that, we've seen buildpack providers upgrade the image on Docker Hub and things Stop Working (even with no code changes on your app). `fly launch` already generates Dockerfiles for [many](https://fly.io/ruby-dispatch/rails-on-docker/) [popular](https://fly.io/docs/elixir/getting-started/#generate-the-app-and-deploy-with-postgres) [frameworks](https://fly.io/docs/django/getting-started/#provision-django).
```

I think Laravel does a Dockerfile too...

Reply:
Our Laravel launcher includes a Dockerfile. It's a bit convoluted to follow, but the Laravel scanner that `fly launch` uses copies files from https://github.com/superfly/flyctl/tree/master/scanner/templates/laravel -- where there's a Dockerfile.